public_html/wp-content/mu-plugins/wcorg-misc.php

```diff
@@ -543,11 +544,31 @@ function send_fatal_to_slack() {
 	return send_error_to_slack( $error['type'], $error['message'], $error['file'], $error['line'] );
 }
 
-if ( ( ! defined( 'WPORG_SANDBOXED' ) || ! WPORG_SANDBOXED ) ) {
+if ( false && ( ! defined( 'WPORG_SANDBOXED' ) || ! WPORG_SANDBOXED ) ) {
 	register_shutdown_function( 'send_fatal_to_slack' );
 	set_error_handler( 'send_error_to_slack', E_ERROR );
 }
 
+/**
+ * Function `send_error_to_slack` above also creates a bunch of files in /tmp/error_limiting folder in order to rate limit the notification.
+ * This function will be used as a cron to clear these error_limiting files periodically.
+ */
+function handle_clear_error_rate_limiting_files() {
+	if ( ! init_error_handling() ) {
+		return;
+	}
+	foreach ( new DirectoryIterator( ERROR_RATE_LIMITING_DIR ) as $file_info ) {
```

Comment:
It'd be nice to have a blank line here, because the two statements are logically distinct. I think it makes the function more readable, similar to how a page of text with paragraphs is much easier to read than one without.